Data/Context/EcommerceContext.cs

```diff
@@ -0,0 +1,101 @@
+﻿using EFCoreConsoleApp.Data.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace EFCoreConsoleApp.Data.Context
+{
+    public class EcommerceContext : DbContext
+    {
+        public EcommerceContext()
+        {
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseNpgsql("Server=127.0.0.1;Port=5432;Database=EFCoreConsole;User Id=postgres;Password=Sancho38293!;");
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Address>()
```

Comment:
Please configure not only relationships between tables but also other properties for all tables e.g. :
entity.HasKey(x => x.Id);
entity.Property(x => x.Prop).IsRequired();
Also don't forget about unique constraints like brand name